rtdata/languages/Deutsch

```diff
@@ -2742,7 +2739,7 @@ TP_LOCALLAB_AUTOGRAYCIE;Automatisch
 TP_LOCALLAB_AVOID;vermeide Farbverschiebungen
 TP_LOCALLAB_AVOIDCOLORSHIFT_TOOLTIP;Passt Farben an den Arbeitsfarbraum an und wendet die Munsell-Korrektur an (Uniform Perceptual Lab).\nMunsell-Korrektur ist deaktiviert wenn Jz oder CAM16 angewandt wird.
 TP_LOCALLAB_AVOIDMUN;Nur Munsell-Korrektur
-TP_LOCALLAB_AVOIDMUN_TOOLTIP;Munsell-Korrektur ist deaktiviert, wenn Jz or CAM16 angewandt wird
+TP_LOCALLAB_AVOIDMUN_TOOLTIP;Munsell-Korrektur ist deaktiviert, wenn Jz or CAM16 angewandt wird.
```

Comment:
Maybe `angewandt`/`angewendet` should be harmonized?

Reply:
you're right, i switched them all to 'angewendet'